blog/content/post/2021-10-28-secure-jtag-on-imx-soc.md

```diff
@@ -0,0 +1,270 @@
+---
+title: 'Securing JTAG on i.MX6 and i.MX8MM'
```

Comment:
Lets try and come up with some more catchy title, maybe something like `Increasing the security of iMX platforms - JTAG fusing`. Test it with https://www.aminstitute.com/headline/. This proposition has ~33%.

Reply:
done